src/main/java/http_server/Response.java

```diff
@@ -0,0 +1,97 @@
+package http_server;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class Response {
```

Comment:
What is the responsibility of this class?

Reply:
Response creates a String server response

- creates the starter response and header
- runs the Route callback which adjusts the response as needed
- finishes response generation and returns the final Response String